packages/contracts/test/dependency-registry/DependencyRegistryV0.test.ts

```diff
@@ -1447,6 +1447,12 @@ describe(`DependencyRegistryV0`, async function () {
         expect(supportedCoreContracts).to.deep.eq([
           config.genArt721Core.address,
         ]);
+
+        const isSupportedCoreContract =
+          await config.dependencyRegistry.isSupportedCoreContract(
+            config.genArt721Core.address
+          );
+        expect(isSupportedCoreContract).to.eq(true);
```

Comment:
nit: should add a case to test the false case

Reply:
Good call out – adding this in the "allows admin to remove supported core contract" case below as an additional check, seems best fit there.